src/awsLambda.re

```diff
@@ -495,3 +495,97 @@ module Sqs = {
 
   type handler = handler_error(Event.t);
 };
+
+module S3 = {
+  module UserIdentity = {
+    [@bs.deriving abstract]
+    type t = {principalId: string};
+
+    let make = t;
+  };
+
+  module RequestParameters = {
+    [@bs.deriving abstract]
+    type t = {sourceIPAddress: string};
+
+    let make = t;
+  };
+
+  module ResponseElements = {
+    [@bs.deriving abstract]
+    type t = {
+      [@bs.as "x-amz-request-id"]
+      xAmzRequestId: string,
+      [@bs.as "x-amz-id-2"]
+      xAmzId2: string,
+    };
+
+    let make = t;
+  };
+
+  module S3Info = {
+    module Bucket = {
+      [@bs.deriving abstract]
+      type t = {
+        name: string,
+        ownerIdentity: UserIdentity.t,
+        arn: string,
+      };
+
+      let make = t;
+    };
+
+    module Object = {
+      [@bs.deriving abstract]
+      type t = {
+        key: string,
+        size: int,
+        eTag: string,
+        versionId: string,
+        sequencer: string,
+      };
+
+      let make = t;
+    };
+
+    [@bs.deriving abstract]
+    type t = {
+      s3SchemaVersion: string,
+      configurationId: string,
+      bucket: Bucket.t,
+      [@bs.as "object"]
+      _object: Object.t,
```

Comment:
The convention in the rest of the file is to use the underscore as in a post-fix position rather than prefix. See all the `type_` occurrences. I tend to think it's better as it doesn't automatically mark the field as unused during pattern matching. Could you change `_object` to `object_`?